addons/website_slides/tests/test_slide_slide.py

```diff
@@ -4,10 +4,30 @@
 import psycopg2
 
 from odoo.addons.website_slides.tests import common as slides_common
+from odoo.addons.website_slides.tests.test_mail import TestSlidesMail
 from odoo.tests.common import users
 from odoo.tools import mute_logger
 
 
+class TestSlidesSlideMail(TestSlidesMail):
+    def test_slide_slide_get_default_recipients(self):
```

Comment:
You don't have to split these by slides model, you're not testing these but the base model. Everything in the TestSlidesChannelMail class would do just fine.

Reply:
I mean `TestSlidesMail`.